terraform/environments/sprinkler/notifications.tf

```diff
@@ -0,0 +1,57 @@
+# Data source to get the ARN of an existing SNS topic
+data "aws_sns_topic" "existing_topic" {
```

Comment:
I wonder if it makes sense to define the sns topic here, rather than in the module (since it's not doing anything there anyway)